src/ui/server/controllers/insurance/business/map-submitted-data/index.ts

```diff
@@ -0,0 +1,62 @@
+import { RequestBody } from '../../../../../types';
+import { objectHasProperty } from '../../../../helpers/object';
+import { FIELD_IDS } from '../../../../constants';
+
+const {
+  EXPORTER_BUSINESS: {
+    COMPANY_HOUSE: { INPUT, COMPANY_NUMBER, COMPANY_INCORPORATED },
+  },
+} = FIELD_IDS.INSURANCE;
+
+/**
+ * maps companyDetails formBody and returns fields in correct format
+ * @param {RequestBody} formBody
+ * @returns {Object} populatedData
+ */
+const mapSubmittedData = (formBody: RequestBody): object => {
+  if (!formBody) {
+    return {};
+  }
```

Comment:
I don't think we need this - formBody is required param and typescript won't allow compilation if this function is called without it 😎